### PR TITLE
DM-31256: Added a tutorial on how to test a CSC at the summit.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: Operational-Procedures/Tutorials/testing-a-csc.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+Overview
+^^^^^^^^
+
+This document decribes how to test a CSC, that is still under development, at the summit.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+- Announcements are to be made in the summit-testing-annoucements Slack channel
```
Comment (tribeiro) [inline Operational-Procedures/Tutorials/testing-a-csc.rst:13]: I hate to tell you but they just renamed the channel to `summit-announce`

File: Operational-Procedures/Tutorials/testing-a-csc.rst [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+would set the DIMM CSC with index 1 OFFLINE.
+
+.. figure:: ./_static/love-configuring.png
+    :name: love-configuring
+
+Connect to the deploy-env container
+-----------------------------------
+
+Once you are sure that no CSC is running in the production environment (anymore), you can login via ssh to azar2.
```
Comment (tribeiro) [inline Operational-Procedures/Tutorials/testing-a-csc.rst:60]: "that no CSC is running" -> "that the CSC you are going to work with is not running"

I hope no one brings the entire system down to debug one CSC... hehehe... 😄

File: Operational-Procedures/Tutorials/testing-a-csc.rst [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+After logging is, the first thing to do is to check and verify that the develop container is running.
```
Comment (tribeiro) [inline Operational-Procedures/Tutorials/testing-a-csc.rst:61]: "After logging is," -> "After logging in to azar2,"

I think you need to specify what "logging in" means, e.g. 

```
ssh azar2.cp.lsst.org
```

**Review by tribeiro — APPROVED** [review]

File: Operational-Procedures/Tutorials/testing-a-csc.rst (hunk 1 above)
Comment (wvreeven) [reply Operational-Procedures/Tutorials/testing-a-csc.rst:13]: Yeah I noticed that. I changed it, also in the `announce-component-usage.rst` file.

File: Operational-Procedures/Tutorials/testing-a-csc.rst (hunk 2 above)
Comment (wvreeven) [reply Operational-Procedures/Tutorials/testing-a-csc.rst:60]: hehe good suggestion!

File: Operational-Procedures/Tutorials/testing-a-csc.rst (hunk 3 above)
Comment (wvreeven) [reply Operational-Procedures/Tutorials/testing-a-csc.rst:61]: Adopted, thanks.

**Pull request merged**